### PR TITLE
Limit which fields PolicyItemsUpdate can change

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/actions/items.go [hunk 1]
```diff
@@ -112,30 +112,41 @@ func itemsUpdate(c buffalo.Context) error {
...
-	if err != nil {
+	item.Name = input.Name
+	item.CategoryID = input.CategoryID
+	item.InStorage = input.InStorage
+	item.Country = input.Country
+	item.Description = input.Description
+	item.Make = input.Make
+	item.Model = input.Model
+	item.SerialNumber = input.SerialNumber
+	item.CoverageStatus = input.CoverageStatus
```
Comment (Baggerone) [inline application/actions/items.go:135]: This shouldn't be included here, should it?

File: application/api/items.go [hunk 2]
```diff
@@ -176,3 +176,47 @@ type ItemStatusInput struct {
...
+	Make string `json:"make"`
+
+	// model
+	Model string `json:"model"`
+
+	// serial number
+	SerialNumber string `json:"serial_number"`
+
+	// coverage status
+	CoverageStatus ItemCoverageStatus `json:"coverage_status"`
```
Comment (Baggerone) [inline application/api/items.go:216]: This shouldn't be included here, right?

**Review by Baggerone — CHANGES_REQUESTED** [review]

**Review by jason-jackson — APPROVED** [review]

File: application/actions/items.go [hunk 3]
```diff
@@ -102,7 +102,7 @@ func itemsCreate(c buffalo.Context) error {
 //     description: policy item create update object
```
Comment (Baggerone) [inline application/actions/items.go:102]: delete `create` (?)

File: application/actions/items.go (hunk 1 above)
Comment (Schparky) [reply application/actions/items.go:135]: I guess I was thinking this endpoint was used to change status, but I forgot about the specialized endpoints for that purpose.

**Review by Baggerone — APPROVED** [review]

**Pull request merged**